app/renderer/src/containers/AppContainer.js

```diff
@@ -17,24 +16,27 @@ const AppContainer = class extends Component {
 
   state = {
     // the current query value
-    q: '',
+    query: '',
     // the current theme
-    theme: '',
+    theme: {},
     // currently pressed keys
     keys: [],
   };
 
   componentDidMount() {
-    ipcRenderer.on(IPC_LOAD_THEME, (evt, theme) => {
-      this.setTheme(theme);
+    ipcRenderer.on(IPC_LOAD_THEME, (evt, nextTheme) => {
+      this.setTheme(nextTheme);
     });
   }
 
   componentDidUpdate() {
-    const { theme } = this.props;
-    if (theme && theme.window && theme.window.width) {
```

Comment:
this also broke, since previously we were picking from props, where as it should be from state.

Another reason not to use destruct on context (props/state)